lib/galaxy/jobs/runners/condor.py

```diff
@@ -57,6 +57,19 @@ def __init__(self, app, nworkers):
         self._init_monitor_thread()
         self._init_worker_threads()
 
+    def __old_state_paths(self, cjs):
+        """For recovery of jobs started prior to standardizing the naming of
+        files in the AsychronousJobState object
+        """
+        if cjs.job_wrapper is not None:
+            job_file = "%s/galaxy_%s.sh" % (self.app.config.cluster_files_directory, cjs.job_wrapper.job_id)
```

Comment:
Maybe do `cluster_files_dir_and_id = (self.app.config.cluster_files_directory, cjs.job_wrapper.get_id_tag())` and then use that in the string formatting below?